lib/Basics/ConditionLocker.h

```diff
@@ -63,7 +63,8 @@ class ConditionLocker {
 /// the condition variable
 #ifdef ARANGODB_SHOW_LOCK_TIME
 
-  ConditionLocker(ConditionVariable* conditionVariable, char const* file, int line);
+  ConditionLocker(ConditionVariable* conditionVariable, char const* file,
+                  int line, bool showLockTime = true);
```

Comment:
This defaults to true to preserve behavior for most instances of `ConditionLocker`. The printing is still gated by a preprocessor flag as well, which is typically not set. In the case that it's set, there is one particular condition used for synchronizing logging which should not be printed, because we then run into issues with logging the logger.